src/App/store/store.ts

```diff
@@ -0,0 +1,11 @@
+import { configureStore } from '@reduxjs/toolkit';
+import userSliceReducer from './userSlice';
+
+export const store = configureStore({
+  reducer: {
+    userSlice: userSliceReducer
+  }
+});
+
```

Comment:
`export const useAppSelector: TypedUseSelectorHook<RootState> = useSelector`

add this also